resources/js/axios.js

```diff
@@ -10,6 +10,13 @@ window.magentoUser = axios.create()
 window.magentoUser.defaults.baseURL = config.magento_url + '/rest/' + config.store_code + '/V1/'
 window.magentoUser.defaults.headers.common['Authorization'] = `Bearer ${localStorage.token}`
 
+window.magentoGraphQL = axios.create()
+window.magentoGraphQL.defaults.baseURL = config.magento_url
+if (localStorage.token) {
+    window.magentoGraphQL.defaults.headers.common['Authorization'] = `Bearer ${localStorage.token}`
```

Comment:
It's better to use token.value from the userStore here, as it allows much more flexibility where we store the token in the future.